Ark.Tools.Outbox/IOutboxContextAsyncFactory.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Ark.Tools.Outbox
+{
+    public interface IOutboxContextAsyncFactory
+    {
+        Task<IOutboxContextAsync> CreateAsync(CancellationToken ctk = default);
```

Comment:
Not sure why this is needed. IContextAsyncFactory is a generic and IContextAsyncFactory<IOutboxContextAsync> is equivalent to this.